include/dd/Package.hpp

```diff
@@ -526,8 +536,10 @@ namespace dd {
         [[nodiscard]] auto& getUniqueTable() {
             if constexpr (std::is_same_v<Node, vNode>) {
                 return vUniqueTable;
-            } else {
+            } else if constexpr (std::is_same_v<Node, mNode>) {
                 return mUniqueTable;
+            } else {
```

Comment:
Consider using another `else if constexpr` for the `else` branch covering density matrices to avoid unexpected situations in the future.